_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-tweetstorm-gather.php

```diff
@@ -55,7 +55,9 @@ public function register_routes() {
 				),
 				'methods'                               => WP_REST_Server::READABLE,
 				'callback'                              => array( $this, 'gather_tweetstorm' ),
-				'allow_blog_token_when_site_is_private' => true,
+				'private_site_security_settings' => [
+					'allow_blog_token_access' => true,
+				],
```

Comment:
We won't be able to commit this as is to Jetpack I'm afraid, because of the PHPCS errors it introduces:

```
----------------------------------------------------------------------
FOUND 1 ERROR AND 5 WARNINGS AFFECTING 5 LINES
----------------------------------------------------------------------
 49 | WARNING | [x] Array double arrow not aligned correctly;
    |         |     expected 27 space(s) between "'args'" and double
    |         |     arrow, but found 34.
    |         |     (WordPress.Arrays.MultipleStatementAlignment.DoubleArrowNotAligned)
 56 | WARNING | [x] Array double arrow not aligned correctly;
    |         |     expected 24 space(s) between "'methods'" and
    |         |     double arrow, but found 31.
    |         |     (WordPress.Arrays.MultipleStatementAlignment.DoubleArrowNotAligned)
 57 | WARNING | [x] Array double arrow not aligned correctly;
    |         |     expected 23 space(s) between "'callback'" and
    |         |     double arrow, but found 30.
    |         |     (WordPress.Arrays.MultipleStatementAlignment.DoubleArrowNotAligned)
 58 | WARNING | [x] Array double arrow not aligned correctly;
    |         |     expected 1 space(s) between
    |         |     "'private_site_security_settings'" and double
    |         |     arrow, but found 8.
    |         |     (WordPress.Arrays.MultipleStatementAlignment.DoubleArrowNotAligned)
 58 | ERROR   | [x] Short array syntax is not allowed
    |         |     (Generic.Arrays.DisallowShortArraySyntax.Found)
 61 | WARNING | [x] Array double arrow not aligned correctly;
    |         |     expected 12 space(s) between
    |         |     "'permission_callback'" and double arrow, but
    |         |     found 19.
    |         |     (WordPress.Arrays.MultipleStatementAlignment.DoubleArrowNotAligned)
----------------------------------------------------------------------
```

This should solve those issues:

```diff
diff --git a/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-tweetstorm-gather.php b/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-tweetstorm-gather.php
index 63ec23b7f..b35bbdc14 100644
--- a/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-tweetstorm-gather.php
+++ b/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-tweetstorm-gather.php
@@ -46,19 +46,19 @@ class WPCOM_REST_API_V2_Endpoint_Tweetstorm_Gather extends WP_REST_Controller {
 			$this->namespace,
 			$this->rest_base,
 			array(
-				'args'                                  => array(
+				'args'                           => array(
 					'url' => array(
 						'description' => __( 'The tweet URL to gather from.', 'jetpack' ),
 						'type'        => 'string',
 						'required'    => true,
 					),
 				),
-				'methods'                               => WP_REST_Server::READABLE,
-				'callback'                              => array( $this, 'gather_tweetstorm' ),
-				'private_site_security_settings' => [
+				'methods'                        => WP_REST_Server::READABLE,
+				'callback'                       => array( $this, 'gather_tweetstorm' ),
+				'private_site_security_settings' => array(
 					'allow_blog_token_access' => true,
-				],
-				'permission_callback'                   => '__return_true',
+				),
+				'permission_callback'            => '__return_true',
 			)
 		);
 	}

```

Reply:
hey sorry I'd missed the notification no this! I made those changes to fix phpcs now :)